test/functional/p2p_orphan_handling.py

```
@@ -0,0 +1,416 @@
+    @property
+    def getdata_received(self):
+        with p2p_lock:
+            return self._getdata_received
+
+    def on_tx(self, message):
+        self._tx_received.append(message)
+
+    def on_getdata(self, message):
+        self._getdata_received.append(message)
+
+    def wait_for_parent_requests(self, txids):
```

Ruff (0.12.2)

Missing return type annotation for public function `wait_for_parent_requests`

Add return type annotation: `None`

(ANN201)